sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/users/UserManagementPage.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.pages.application.users;
+
+import org.openqa.selenium.By;
+
+public class UserManagementPage {
+  public static final By NEW_USER_BUTTON = By.id("userNewUser");
+  public static final By SEARCH_USER_INPUT = By.id("search");
+  public static final By USER_NAME_GRID_CELL = By.cssSelector("tr td:nth-child(5)");
+  public static final By FIRST_EDIT_BUTTON_FROM_LIST =
```

Comment:
this is not the best approach, because sometimes data is not displayed in the same order. I suggest identifying a specific div based on an unique ID or something, and then go to the edit button. 
You can use this as a reference
public static final String EDIT_BUTTON_XPATH_BY_TEXT =
      "//td[contains(text(),'%s')]/../td/span[contains(@class, 'v-icon-edit')]";

Reply:
I searched for username witch is unique. So, if will be more users at search it will be a bug :D.